File: Ser2023/Company/Company.Web/Modules/ServerTypes/Store/ShippingState.ts
import { Decorators } from "@serenity-is/corelib";

export enum ShippingState {
    NotShipped = 0,
    Shipped = 1
}
Decorators.registerEnumType(ShippingState, 'Company.Store.ShippingState', 'Company.ShippingState');

File: Ser2023/Company/Company.Web/Modules/Store/Shippers/ShippingState.cs

namespace Company.Store
{
    using Serenity.ComponentModel;
    using System.ComponentModel;

    [EnumKey("Company.ShippingState")]
    public enum ShippingState
    {
        [Description("Not Shipped")]
        NotShipped = 0,
        [Description("Shipped")]
        Shipped = 1
    }
}